### DISCUSSION TITLE
how to use meridiemHour? I can't find any info in docs

### DISCUSSION THREAD

**Discussion by LiuTing0403** [moment/moment#6412] [Q&A] [answered]
I found meridiemHour is defined in zh-cn locale. It looks that when hour is 12, should return 0. 
```
        meridiemHour: function (hour, meridiem) {
            if (hour === 12) {
                hour = 0;
            }
            if (meridiem === '凌晨' || meridiem === '早上' || meridiem === '上午') {
                return hour;
            } else if (meridiem === '下午' || meridiem === '晚上') {
                return hour + 12;
            } else {
                // '中午'
                return hour >= 11 ? hour : hour + 12;
            }
        },
```
But, I always got '凌晨12:30' when format. 
`moment('2024-08-22 00:30').locale('zh-cn').format('Ah:mm')
`
How to format to  '凌晨00:30'?

**Comment by mattjohnsonpint** [comment, marked as answer]
Thanks for asking! `meridiemHour` is a locale hook used while parsing meridiem text; it does not control output formatting. For the requested zero-padded 24-hour value alongside the Chinese meridiem, I'd use `moment(value).locale('zh-cn').format('AHH:mm')`, which produces output such as `凌晨00:30`.
